packages/react-components/src/components/AttachmentCard.tsx

```diff
@@ -116,13 +80,13 @@ export const _AttachmentCard = (props: _AttachmentCardProps): JSX.Element => {
 
   useEffect(() => {
     if (showProgressIndicator) {
-      setAnnouncerString(`${uploadStartedString} ${attachmentName}`);
+      setAnnouncerString(`${uploadStartedString} ${attachment.name}`);
```

Comment:
Unrelated issue to this PR, it looks like this string substitution is assuming positioning of the wording based on English.

Reply:
i am not sure, we can have an a11y task to look into this